R/nma.R

```diff
@@ -708,6 +714,22 @@ nma <- function(network,
       }
     }
 
+    if (".mu" %in% all.vars(regression)) {
+      if (".mu" %in% colnames(idat_agd_arm)) {
+        warn(c(
+          "Detected `.mu` in the `regression` formula and in the data.",
+          i = "`.mu` is a special variable refering to the modelled baseline risk."
+        ))
+      }
+
+      ref_trt <- levels(idat_agd_arm$.trt)[1L]
+
+      idat_agd_arm <- idat_agd_arm %>%
+        dplyr::group_by(.data$.study) %>%
+        dplyr::mutate(.mu = as.integer(.data$.trt != ref_trt & ref_trt %in% .data$.trt)) %>%
```

Comment:
Do we want instead to put a dummy variable for .mu in every non-reference treatment arm?
```suggestion
        dplyr::mutate(.mu = as.integer(.data$.trt != ref_trt)) %>%
```
And then let interactions with treatment be governed by the setting of `class_interactions`?